src/script_lua.c

```diff
@@ -1432,52 +1434,69 @@ void luaRegisterLogFunction(lua_State *lua) {
     lua_settable(lua, -3);
 }
 
+/*
+ * Adds server.* functions/fields to lua such as server.call etc.
+ * This function only handles fields common between Functions and LUA scripting.
+ * scriptingInit() and functionsInit() may add additional fields specific to each.
+ */
 void luaRegisterServerAPI(lua_State *lua) {
+    /* In addition to registering server.call/pcall API, we will throw a customer message when a script accesses
```

Comment:
`customer message` sounds weird.

Reply:
good catch, it is custom message - fixed.